woocommerce/woocommerce-android
# Composite products analytics

Closes: #8611

### Description
This adds an analytics event when the Components row is tapped in product details (for [Composite Products](https://woocommerce.com/products/composite-products/)):

* `product_details_view_components_tapped ` (Event [registration](https://github.com/Automattic/tracks-events-registration/pull/1558))

### Testing instructions

#### Prerequisites

1. Install and activate the [Composite Products](https://woocommerce.com/products/composite-products/) on your store.
2. Create at least one product with the Composite product type, with at least one component.

#### To test

1. Build and run the app.
2. Open the Products tab and select a composite product.
3. Tap the Components row in the product details.
4. Confirm the app navigates to the Components list.
5. Confirm the new event is triggered.

### Images/gif
```
🔵 Tracked: product_details_view_components_tapped, Properties: {"blog_id":214253715,"is_wpcom_store":true,"is_debug":true}
```
